library/Icinga/Application/Modules/DashboardContainer.php

```diff
@@ -35,7 +35,11 @@ public function setDashlets(array $dashlets)
      */
     public function getDashlets()
     {
-        return $this->dashlets ?: array();
+        uasort($this->dashlets, function (array $x, array $y) {
+            return $x['priority'] - $y['priority'];
```

Comment:
You should return a -1, 0 or 1 here, as usually expected. There's also the spaceship operator since php 7 which simplifies this: `<=>`